android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -55,57 +48,38 @@ public ExoPlayerView(Context context) {
 
         this.context = context;
 
+        shutterView = findViewById(androidx.media3.ui.R.id.exo_shutter);
+
         layoutParams = new ViewGroup.LayoutParams(
                 ViewGroup.LayoutParams.MATCH_PARENT,
                 ViewGroup.LayoutParams.MATCH_PARENT);
 
         componentListener = new ComponentListener();
-
         FrameLayout.LayoutParams aspectRatioParams = new FrameLayout.LayoutParams(
                 FrameLayout.LayoutParams.MATCH_PARENT,
                 FrameLayout.LayoutParams.MATCH_PARENT);
         aspectRatioParams.gravity = Gravity.CENTER;
         layout = new AspectRatioFrameLayout(context);
         layout.setLayoutParams(aspectRatioParams);
 
-        shutterView = new View(getContext());
-        shutterView.setLayoutParams(layoutParams);
-        shutterView.setBackgroundColor(ContextCompat.getColor(context, android.R.color.black));
-
-        subtitleLayout = new SubtitleView(context);
-        subtitleLayout.setLayoutParams(layoutParams);
-        subtitleLayout.setUserDefaultStyle();
-        subtitleLayout.setUserDefaultTextSize();
-
         updateSurfaceView(viewType);
-
-        adOverlayFrameLayout = new FrameLayout(context);
-
-        layout.addView(shutterView, 1, layoutParams);
-        if (localStyle.getSubtitlesFollowVideo()) {
```

Comment:
this getSubtitlesFollowVideo is not used anymore, so you removed a recent feature ...